app/src/userpages/components/StreamPage/List/index.jsx

```diff
@@ -122,7 +124,7 @@ const StreamList = () => {
         setSort,
         resetFilter,
     } = useFilterSort(sortOptions)
-    const [dialogTargetStream, setDialogTargetStream] = useState(undefined)
+    const [dialogTargetStream, setDialogTargetStream]: TargetStreamSetter = useState(null)
```

Comment:
I wish we could just stop using `null` but I see the rest of this code uses `null` so 🤷‍♀